ui/cypress/e2e/marketplace.cy.ts

```diff
@@ -0,0 +1,28 @@
+import getMarketplaceListings from '@/lib/subscription/getMarketplaceListings'
```

Comment:
Could we also test the inverse? E.g. a signed in Citizen should see this without the upcharge